app/styles/front.scss

```diff
@@ -347,17 +347,7 @@ $autre-lighter: #cce4e7;
 
```

Comment:
Yes, there are many. This is indeed extremely ugly.

The basic is to have a “global” / “main” CSS file, then additional ones for each screen.

When you have components though, it makes sense to have a file for each component.

We need a destination, and will refactor along along the way.